### ISSUE TITLE
i found this error during the scroll

### ISSUE THREAD

**Issue by jjhesk** [twotoasters/JazzyListView#32]
This bug is not occur often but only when the list goes really long and fast.

```
03-02 18:17:32.403  15537-15537/com.buildsystemexample.app.test_native E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: com.buildsystemexample.app.test_native, PID: 15537
    java.lang.NullPointerException
            at com.nineoldandroids.view.ViewPropertyAnimatorICS.<init>(ViewPropertyAnimatorICS.java:22)
            at com.nineoldandroids.view.ViewPropertyAnimator.animate(ViewPropertyAnimator.java:62)
            at com.twotoasters.jazzylistview.JazzyHelper.doJazzinessImpl(JazzyHelper.java:195)
            at com.twotoasters.jazzylistview.JazzyHelper.doJazziness(JazzyHelper.java:187)
            at com.twotoasters.jazzylistview.JazzyHelper.onScroll(JazzyHelper.java:103)
            at android.widget.AbsListView.invokeOnItemScrollListener(AbsListView.java:1417)
            at android.widget.GridView.layoutChildren(GridView.java:1305)
            at android.widget.AbsListView$FlingRunnable.run(AbsListView.java:4209)
            at android.view.Choreographer$CallbackRecord.run(Choreographer.java:761)
            at android.view.Choreographer.doCallbacks(Choreographer.java:574)
            at android.view.Choreographer.doFrame(Choreographer.java:543)
            at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)
            at android.os.Handler.handleCallback(Handler.java:733)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:136)
            at android.app.ActivityThread.main(ActivityThread.java:5146)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:515)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:732)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:566)
            at dalvik.system.NativeStart.main(Native Method)

```

**Comment by jbarr21** [comment]
does this still happen for you on v1.2? can you please try that and see if that fixes the issue?

**Comment by sbaar** [comment]
This happens to me very occasionaly usually on our users' older phones but i can't upgrade to 1.2 because I keep my minsdk at 14.

**Comment by jbarr21** [comment]
@sbaar ah, ok. i should be able to bring this down to 15. there's like no users running 14, so u should really bump ur app up to 15, but let me look at an update

**Comment by jbarr21** [comment]
@sbaar just pushed an update to 1.2.1 it should be available shortly on jcenter or prolly tmr morning on mavenCentral. lemme know if that fixes your issue

**Comment by jjhesk** [comment]
@sbaar would you issue a gradle repo on jcenter?

**Comment by sbaar** [comment]
@jjhesk jbarr21 did

https://bintray.com/twotoasters/maven/com.twotoasters.jazzylistview/1.2.1/view

**Comment by jjhesk** [comment]
@sbaar  i am using API 21 to compile and min sdk is 16. I had to import the library into the project since there is no gradle path to include. Here is my setup code in gradle

```gradle
apply plugin: 'android-library'

android {
    compileSdkVersion 20
    buildToolsVersion '21.1.2'

    defaultConfig {
        minSdkVersion 16
        targetSdkVersion 21
    }
    sourceSets {
        main {
            manifest.srcFile 'AndroidManifest.xml'
            java.srcDirs = ['src']
            resources.srcDirs = ['src']
            aidl.srcDirs = ['src']
            renderscript.srcDirs = ['src']
            res.srcDirs = ['res']
            assets.srcDirs = ['assets']
        }

        instrumentTest.setRoot('tests')

        debug.setRoot('build-types/debug')
        release.setRoot('build-types/release')
    }
}

dependencies {
    compile 'com.android.support:support-v4:19.0.0'
    compile 'com.nineoldandroids:library:2.4.0'
}

//apply from: 'https://raw.github.com/twotoasters/gradle-mvn-push/master/gradle-mvn-push.gradle'
```

**Comment by jjhesk** [comment]
@sbaar noticed.

**Comment by sbaar** [comment]
I don't quite understand what the issue you're having is. It's available on maven central right now. compile 'com.twotoasters.jazzylistview:library:1.2.1'
http://search.maven.org/#artifactdetails%7Ccom.twotoasters.jazzylistview%7Clibrary%7C1.2.1%7Caar

**Comment by jjhesk** [comment]
lets add this path on the readme

**Comment by jbarr21** [comment]
Resolved by #28
